src/main/java/edu/harvard/iq/dataverse/search/IndexServiceBean.java

```diff
@@ -183,11 +183,11 @@ public void close() {
         if (dataverse.isReleased()) {
             solrInputDocument.addField(SearchFields.PUBLICATION_STATUS, PUBLISHED_STRING);
             solrInputDocument.addField(SearchFields.RELEASE_OR_CREATE_DATE, dataverse.getPublicationDate());
-            solrInputDocument.addField(SearchFields.RELEASE_OR_CREATE_DATE_SEARCHABLE_TEXT, convertToFriendlyDate(dataverse.getPublicationDate()));
+            //solrInputDocument.addField(SearchFields.RELEASE_OR_CREATE_DATE_SEARCHABLE_TEXT, convertToFriendlyDate(dataverse.getPublicationDate()));
```

Comment:
@JayanthyChengan can you just remove these lines instead of leaving commented out? (the constant and methods they depend on are already removed)

Reply:
yes, removed and committed the code